cloudevents/sdk/event/attributes/specversion_attribute.py

```diff
@@ -0,0 +1,23 @@
+#  Copyright 2018-Present The CloudEvents Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from enum import Enum
+
+
+class SpecVersion(Enum):
+    v0_3 = "0.3"
+    v1_0 = "1.0"
```

Comment:
I was thinking of having a string enum. This way it should be possible to use it interchangeably with a simple string.
```suggestion
class SpecVersion(str, Enum):
    v0_3 = "0.3"
    v1_0 = "1.0"
```